src/Native/test/test_ntt_binary_max.cpp

```diff
@@ -709,28 +709,36 @@ TEST(BinaryTestMaxFloat, ranked_ranked_ranked_broadcast_multidirectional) {
     EXPECT_TRUE(NttTest::compare_tensor(*ntt_output1, *ntt_output2));
 }
 
-TEST(BinaryTestMaxFloat, vecotor_8) {
-    // init
-    ntt::vector<float, 8> ntt_lhs, ntt_rhs;
-    NttTest::init_tensor(ntt_lhs, -10.f, 10.f);
-    NttTest::init_tensor(ntt_rhs, -10.f, 10.f);
-
-    // ntt
+template <typename T, size_t vl> void test_vector() {
+    ntt::vector<T, vl> ntt_lhs, ntt_rhs;
+    NttTest::init_tensor(ntt_lhs, static_cast<T>(-10), static_cast<T>(10));
+    NttTest::init_tensor(ntt_rhs, static_cast<T>(-10), static_cast<T>(10));
     auto ntt_output1 = ntt::max(ntt_lhs, ntt_rhs);
-
-    // ort
     auto ort_lhs = NttTest::ntt2ort(ntt_lhs);
     auto ort_rhs = NttTest::ntt2ort(ntt_rhs);
     ortki::OrtKITensor *ort_array[] = {ort_lhs, ort_rhs};
     auto ort_output =
         ortki_Max(ort_array, sizeof(ort_array) / sizeof(ort_array[0]));
```

Comment:
It's better to use `std::size` to replace `sizeof(ort_array) / sizeof(ort_array[0])`